components/automate-chef-io/content/docs/install.md

```diff
@@ -166,9 +168,10 @@ Add the following to your config.toml:
 
 Because externally-deployed Elasticsearch nodes will not have access to Automate's built-in backup storage services, you must configure Elasticsearch backup settings separately from Automate's primary backup settings. You can configure backups to use either the local filesystem or S3.
 
-##### Backup to Local Filesystem
+##### Backup Externally-Deployed Elasticsearch to Local Filesystem
```

Comment:
@ kagarmoe These directions are included on the install page because

1. They are required for a user who is backing up an installation that includes external elasticsearch, and
2. The goal of the basic install is to get the users to an installation with sensible defaults, one of which is that it can be backed up.